official/resnet/resnet_run_loop.py

```diff
@@ -72,6 +75,16 @@ def process_record_dataset(dataset, is_training, batch_size, shuffle_buffer,
   # dataset for the appropriate number of epochs.
   dataset = dataset.repeat(num_epochs)
 
+  if is_training and num_gpus is not None and num_gpus > 0:
+    total_examples = num_epochs * examples_per_epoch
```

Comment:
do you need to also check that examples_per_epoch is not None?

Reply:
I don't believe so, because it will only be None during evaluation. But I suppose it doesn't hurt to add it.